src/components/NcPopover/NcPopover.vue

```diff
@@ -442,7 +445,18 @@ $arrow-width: 10px;
 		left: 0;
 		display: block !important;
 
-		filter: drop-shadow(0 1px 10px var(--color-box-shadow));
+		.v-popper__wrapper {
+			/*
+			 * In theory, "filter: drop-shadow" would look better here with arrow shadow.
+			 * In fact, in results in a blurry popover in Chromium on scaling.
```

Comment:
Typical Blink engine issue, their hardware acceleration is good - sometimes - and really really bad in most cases. Gecko is buggy but at least consistent as they wrote the renderer themself.